dotfiles/prompt

```diff
@@ -160,3 +160,6 @@ _prompt() {
 }
 
 PROMPT_COMMAND='_prompt'
+if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
+  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
```

Comment:
Okay. But then does Apple terminal become a dependency of buffet? I am okay with this, but I'd like it to be documented and checked for in the install script. 



> On Jun 6, 2015, at 2:29 AM, bendy-build-butler <notifications@github.com> wrote:
> 
> In dotfiles/prompt:
> 
> > @@ -160,3 +160,6 @@ _prompt() {
> >  }
> >  
> >  PROMPT_COMMAND='_prompt'
> > +if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
> > +  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
> I think it's built in actually, just only available in Apple Terminal and not iTerm.
> …
> —
> Reply to this email directly or view it on GitHub.
>